type.go

```diff
@@ -59,6 +69,9 @@ func (ht *HsTime) UnmarshalJSON(b []byte) error {
 // String implemented Stringer.
 // If the value is zero, it will be displayed as `<nil>`.
 func (ht *HsTime) String() string {
+	if ht == nil {
+		return "<nil>"
```

Comment:
Why don't you just return "" or "nil" but "<nil>"?

Reply:
Certainly..
I have been doing this since the original implementation, but it looks like an empty string is fine.

Reply:
Fixed to return `nil` if the value is null, or empty string if the value is zero